lib/LightPanel/LightPanel.js

```diff
@@ -149,12 +149,23 @@ module.exports = kind(
 		if (!this.$.client.children.length) {
 			this.createComponents(this.clientComponents);
 			this.$.client.render();
-			this.$.client.addClass('populated');
+			this.didClientRender();
 		}
 
-		if (!Spotlight.getCurrent()) {
+		if (!Spotlight.getPointerMode() && !Spotlight.isSpottable(Spotlight.getCurrent(), true)) {
```

Comment:
Hi @ryanjduffy , Sorry for the delayed response.
I don't remember the exact scenario because it was done while I modify huge amount of code on BreadcrumbSupport.js. I think we can skip this spotlight change in this PR for now. So, we can create separate PR if it is needed.